plugins/module_utils/crypto/cryptography_support.py

```diff
@@ -68,24 +69,53 @@ def cryptography_get_extensions_from_cert(cert):
     # Since cryptography won't give us the DER value for an extension
     # (that is only stored for unrecognized extensions), we have to re-do
     # the extension parsing outselves.
+    backend = default_backend()
+    try:
+        # For certain old versions of cryptography, backend is a MultiBackend object,
+        # which has no _lib attribute. In that case, revert to the old approach.
+        x = backend._lib
```

Comment:
Did you mean 
```suggestion
        backend._lib
```

Reply:
Both should have the same result. But yeah, removing that assignment is probably a good idea. I did it in all places at once in 5a27b231214e318156d79220df3ea89347879377.